execution_engine_testing/tests/src/test/system_contracts/auction/bids.rs

```diff
@@ -4129,3 +4130,325 @@ fn should_allow_delegations_with_minimal_floor_amount() {
 
     builder.exec(delegation_request_2).expect_success().commit();
 }
+
+#[ignore]
+#[test]
+fn should_enforce_max_delegators_per_validator_cap() {
```

Comment:
Can you extend this test a little?

What I think of is to test that if a validator is fully saturated with delegators, we can remove one of the validators and be able to add another one in it's place.

Reply:
Extended test